browser/brave_news/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+import("//brave/build/config.gni")
```

Comment:
I deleted this file that I created a while ago, it's replaced with `sources.gni`